src/outline.jl

```diff
@@ -326,8 +330,12 @@ function str_value(x)
         return string("\"\"\"", x.val, "\"\"\"")
     elseif x.kind === CSTParser.Tokens.STRING
         return string("\"", x.val, "\"")
+    elseif x.kind === CSTParser.Tokens.EQ
+        return " = "
+    elseif x.kind === CSTParser.Tokens.WHERE
+        return " where "
     elseif x.typ === CSTParser.Parameters
-        return ";" * join(str_value(a) for a in x)
+        return "; " * join(str_value(a) for a in x)
```

Comment:
I made these changes since strings from `str_value` now can be shown in ink.goto.goto items, and it would make more consistent with the strings rendered `Method`s.

But you mayn't like this, in especially in outline.
If so, I will revert this change, so let me know.

Reply:
Let's keep it, I've been annoyed by this too. I'm sure there must be a "correct" way to handle whitespace, but this is good enough for me.

Reply:
Let's keep it, I've been annoyed by this too. I'm sure there must be a "correct" way to handle whitespace, but this is good enough for me.

Reply:
okay, then let's leave implementing in a "correct" way for the future :p